### PR TITLE
RavenDB-14856 Allow to import attachments, counters, time series & re…

### PR DESCRIPTION
**Pull request by Danielle9897**
…visions without documents

### CODE REVIEW

File: src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts [hunk 1]
```diff
@@ -105,6 +93,37 @@ class exportDatabaseModel {
...
+                }
+                if (this.includeRevisionDocuments()) {
+                    items.push(["Revisions"]);
+                }
+            }
+
+            switch (items.length) {
+                case 4:
+                case 3:
+                    return `${items.slice(0, items.length-1).join(', ')} & ${items[items.length-1]}`;
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts]: maybe move this to default case to support not only length of 0, 1,2,3,4

case 0:
 return ""

File: src/Raven.Studio/typescript/models/database/tasks/importDatabaseModel.ts [hunk 2]
```diff
@@ -98,7 +85,38 @@ class importDatabaseModel {
...
+                }
+                if (this.includeTimeSeries()) {
+                    items.push(["Time Series"]);
+                }
+                if (this.includeRevisionDocuments()) {
+                    items.push(["Revisions"]);
+                }
+            }
+
+            switch (items.length) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/tasks/importDatabaseModel.ts]: or maybe let's move this to generalIUtils? as right now we duplicate code + we can support all lengths.

Comment (ml054) [reply src/Raven.Studio/typescript/models/database/tasks/importDatabaseModel.ts]: or... can we simply join with comma?

File: src/Raven.Studio/wwwroot/App/views/database/tasks/exportDatabase.html [hunk 3]
```diff
@@ -13,74 +13,80 @@ <h3>Export your database to a .ravendbdump file</h3>
...
-                            <input id="toggleDocuments" type="checkbox" data-bind="checked: includeDocuments" />
-                            <label for="toggleDocuments">Include Documents</label>
-                        </div>
-                        <div class="margin-left margin-left-lg">
-                            <div class="toggle">
-                                <input id="toggleAttachments" type="checkbox"
-                                       data-bind="checked: includeAttachments" />
-                                <label for="toggleAttachments">Include Attachments</label>
-                            </div>
+                        <h4>Documents and Extentsions:</h4>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/tasks/exportDatabase.html]: typo?

File: src/Raven.Studio/wwwroot/App/views/database/tasks/importDatabaseFromFile.html [hunk 4]
```diff
@@ -31,97 +31,102 @@ <h3>Import data from a .ravendbdump file into the current database</h3>
...
-                                <input id="importAttachments" type="checkbox"
-                                       data-bind="checked: includeAttachments" />
-                                <label for="importAttachments">Include Attachments</label>
-                            </div>
-                            <div class="toggle">
-                                <input id="importLegacyAttachments" type="checkbox"
-                                       data-bind="checked: includeLegacyAttachments" />
-                                <label for="importLegacyAttachments">Include Legacy Attachments</label>
-                            </div>
+                        <h4>Documents and Extentsions:</h4>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/tasks/importDatabaseFromFile.html]: typo

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/tasks/importDatabaseFromFile.html]: oops - fixed

File: src/Raven.Studio/wwwroot/App/views/database/tasks/exportDatabase.html (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/tasks/exportDatabase.html]: oops - fixed

File: src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/tasks/exportDatabaseModel.ts]: done

File: src/Raven.Studio/typescript/models/database/tasks/importDatabaseModel.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/tasks/importDatabaseModel.ts]: done - moved to generalIUtils
but I prefer to keep it grammatically correct w/ '&' before the last one

**Review by ml054 — APPROVED** [review]

**Pull request merged**